kotlin-analysis-api/src/main/kotlin/com/google/devtools/ksp/impl/symbol/kotlin/KSTypeReferenceImpl.kt

```diff
@@ -30,8 +30,7 @@ import org.jetbrains.kotlin.analysis.api.annotations.annotations
 import org.jetbrains.kotlin.analysis.api.types.KtType
 
 class KSTypeReferenceImpl(private val ktType: KtType) : KSTypeReference {
-    override val element: KSReferenceElement?
-        get() = TODO("Not yet implemented")
+    override val element: KSReferenceElement? = null
```

Comment:
How do processors get the element then?

Reply:
I forgot to put a TODO here, this needs to be addressed separately.